### ISSUE TITLE
After torch.cat(input_ids, dim=1) in data_qwen_packed.py, the token_length of input_ids is very likely larger than max_token_length if batch_size is large than 1

### ISSUE THREAD

**Issue by yucheng-zzhao** [QwenLM/Qwen2.5-VL#1230]
have not find how the training code deal with this case.
the torch.cat(input_ids, dim=1) is from PackedDataCollatorForSupervisedDataset function in data_qwen_packed.py file.
